hotspot/src/main/java/io/deephaven/hotspot/HotSpotImpl.java

```diff
@@ -0,0 +1,19 @@
+package io.deephaven.hotspot;
+
+import com.google.auto.service.AutoService;
+import io.deephaven.util.hotspot.HotSpot;
+import sun.management.ManagementFactoryHelper;
+
+@AutoService(HotSpot.class)
+public class HotSpotImpl implements HotSpot {
+
+    @Override
+    public long getSafepointCount() {
+        return ManagementFactoryHelper.getVMManagement().getSafepointCount();
+    }
+
+    @Override
+    public long getTotalSafepointTimeMillis() {
+        return ManagementFactoryHelper.getVMManagement().getTotalSafepointTime();
+    }
```

Comment:
How should we choose to fail when the user doesn't add `add-opens`? As written now, the failure will occur at method-call time. It may be better to capture `getVMManagement()` during the construction of `HotSpotImpl`.